config/locales/ja.yml

```diff
@@ -45,7 +45,7 @@ ja:
     accessibility:
       full_help_html: 'お客様が支援技術（スクリーンリーダーなど）を使用していて、このドキュメントよりアクセスしやすい形式のバージョンが必要な場合は、必要なフォーマットや使用している支援技術を添え％{email}にEメールをお送くりください。
 
-'
+        '
```

Comment:
whitespace

Reply:
Looks like this was added in when I normalized the locale files.  I'll remove the whitespace here and add the normalization as either another PR or another commit.